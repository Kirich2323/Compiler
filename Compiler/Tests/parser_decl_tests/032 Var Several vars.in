var
    a : float;
    b : integer;