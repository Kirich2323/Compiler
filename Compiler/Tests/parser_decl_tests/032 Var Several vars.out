main
----------
char                     type 
float                    type 
integer                  type 
a                         var           float
b                         var         integer